jwst/stpipe/tests/test_step.py

```diff
@@ -119,6 +121,9 @@ def test_saving_pars(tmpdir):
     saved = datamodels.StepParsModel(str(saved_path))
     assert saved.parameters == ParsModelWithPar3.parameters
 
+    step.closeout()
+    saved.close()
```

Comment:
Instead of `saved.close()` I would use a `with` context manager:

```python
with datamodels.StepParsModel(str(saved_path)) as saved:
    assert saved.parameters == ParsModelWithPar3.parameters
```